data_engineer/README.md

```diff
@@ -1,22 +1,17 @@
-## Data Python Exercise
+## Data Engineer Python Exercise  
 
-This exercise is designed to show us your python programming skills. 
-Although the problem isn't very hard, we would like you to tackle the task the same way you would with a client. 
-We expect that you provide us with a python application that follows python best practices as defined by the [README](../README.md) at the root of the repository.   
+For this exercise, please write a Python application that processes 
+two data files:
 
-Your first tasks are to :  
+1. `students.csv` containing student data
+2. `teachers.parquet` containing teacher data
 
-* Clone this repo
-* Un-encrypt the project direction file  ```interview_problem.md.encrypted```. The file has been encrypted using the AES-256-CBC algorithm with openssl.  
+From these files, generate an output file in json listing:
 
-* Follow the directions interview_problem.md
+1. Each student
+2. The teacher the student has
+3. he class ID the student is scheduled for
 
-### Here are some resources on openssl   
-https://wiki.openssl.org/index.php/Enc
```

Comment:
My preference would be to not have remove the requirement to decrypt the directions. It solves two issues.
1. A basic task of following the copy and past directions.
2. Having recruiting exercises free out in the open isn't the smartest strategy.

Reply:
These aren't unreasonable goals, but we're looking to hire substantially higher in the food chain than "can copy and paste commands." We run the risk of losing good people if we ask them to do do annoyingly basic stuff. 

If we want to keep the exercises secret, there are other ways to do that.

Reply:
I don't think I feel comfortable being on the PR, since I am not involved in the hiring process at this time.